### PR TITLE
Adjust CSS to better match mocks

### PR DESCRIPTION
**Pull request by bemasc**

### CODE REVIEW

**Comment by bemasc** [comment]
New appearance:
Screenshot from 2021-08-04 16-36-44

File: src/server_manager/web_app/ui_components/outline-region-picker-step.ts [hunk 1]
```diff
@@ -152,17 +153,15 @@ export class OutlineRegionPicker extends LitElement {
...
-                ${option.markedLowCost ? this.localize('region-lowest-cost') : ''}
-              </div>
-            </div>
             <div class="flag-overlay">
               <img class="flag" src="${this._flagImage(option)}">
             </div>
             <div class="geo-name">${getShortName(option.cloudLocation, this.localize)}</div>
             <div class="country-name">${option.cloudLocation.location?.countryIsRedundant() ? '' :
                 localizeCountry(option.cloudLocation.location, this.language)}</div>
+            ${option.markedLowCost ?
```
Comment (fortuna) [inline src/server_manager/web_app/ui_components/outline-region-picker-step.ts:162]: Please rename the fields in code to reflect the new wording.

**Review by fortuna — CHANGES_REQUESTED** [review]
Why are there two regions selected in the screenshot?

File: src/server_manager/web_app/ui_components/outline-region-picker-step.ts (hunk 1 above)
Comment (bemasc) [reply src/server_manager/web_app/ui_components/outline-region-picker-step.ts]: Done

**Review by fortuna — APPROVED** [review]
Thanks!

**Pull request merged**